molo/core/middleware.py

```diff
@@ -114,12 +114,7 @@ def process_response(self, request, response):
 class SetSiteMiddleware(django.utils.deprecation.MiddlewareMixin):
     """Sets the language code"""
```

Comment:
Please change the comment here to indicate that this method caches the site in the request.